dns_server/src/config.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::Parser;
+use directories::BaseDirs;
+
+#[derive(Parser, Debug)]
+pub struct DnsServerConfig {
+    /// The path to the data directory
+    #[clap(long, default_value_t = default_data_dir())]
+    pub datadir: String,
+
+    /// Use testnet
+    #[clap(long)]
+    pub testnet: bool,
```

Comment:
Please don't use boolean for this. We may have to instantiate multiple testnets. The right thing here is to use network ids that parse from strings. So "testnet" is testnet, and "mainnet" is mainnet. The default is mainnet.

Reply:
OK, updated. `clap::ValueEnum` worked nicely.